cli/popper/gha.py

```diff
@@ -251,32 +254,44 @@ def execute(self, cmd, log_tag):
         out_fname = os.path.join(self.log_path, log_tag + '.out')
         err_fname = os.path.join(self.log_path, log_tag + '.err')
 
-        with open(out_fname, "w") as outf, open(err_fname, "w") as errf:
-            p = subprocess.Popen(cmd, stdout=outf, stderr=errf, shell=True,
-                                 preexec_fn=os.setsid)
+        if self.verbose:
+            rc = pu.exec_cmd_verbose(cmd)[0]
```

Comment:
this will result in the log files not being created, which is not the expected behavior. Instead, we would like a way of streaming to both, `stdout` and the log files.